bentoctl/cli/__init__.py

```diff
@@ -144,19 +144,13 @@ def build(
     deployment_config.set_bento(bento_tag)
     local_docker_tag = deployment_config.generate_local_image_tag()
     debug_mode = get_debug_mode()
-    with TempDirectory(cleanup=debug_mode) as dist_dir:
-        (
-            dockerfile_path,
-            dockercontext_path,
-            build_args,
-        ) = deployment_config.create_deployable(
+    with TempDirectory(debug_mode=debug_mode) as dist_dir:
+        dockercontext_path = deployment_config.create_deployable(
```

Comment:
It seems to me we just need to create a tempdirectory to create files, then copy over to the target deployment.

Using `fs` here would also solve the issue.

Reply:
yeah, using moving over to fs will improve some of our file handling usecases. Will add that

Reply:
using fs is probably not necessary here